plugins/modules/web_infrastructure/django_manage.py

```diff
@@ -57,41 +67,66 @@
     type: bool
   database:
     description:
-      - The database to target. Used by the C(createcachetable), C(flush), C(loaddata), C(syncdb), and C(migrate) I(command).
+      - The database to target. Used by the C(createcachetable), C(flush), C(loaddata), C(syncdb),
+        and C(migrate) commands.
+    type: str
     required: false
   failfast:
     description:
-      - Fail the command immediately if a test fails. Used by the 'test' command.
+      - Fail the command immediately if a test fails. Used by the C(test) command.
     required: false
-    default: "no"
+    default: false
     type: bool
+    aliases: [fail_fast]
   fixtures:
     description:
-      - A space-delimited list of fixture file names to load in the database. B(Required) by the 'loaddata' command.
+      - A space-delimited list of fixture file names to load in the database. B(Required) by the C(loaddata) command.
+    type: str
     required: false
   skip:
     description:
-     - Will skip over out-of-order missing migrations, you can only use this parameter with I(migrate)
+     - Will skip over out-of-order missing migrations, you can only use this parameter with C(migrate) command.
     required: false
     type: bool
   merge:
     description:
-     - Will run out-of-order or missing migrations as they are not rollback migrations, you can only use this parameter with 'migrate' command
+     - Will run out-of-order or missing migrations as they are not rollback migrations, you can only use this
+       parameter with C(migrate) command
     required: false
     type: bool
   link:
     description:
-     - Will create links to the files instead of copying them, you can only use this parameter with 'collectstatic' command
+     - Will create links to the files instead of copying them, you can only use this parameter with
+       C(collectstatic) command
     required: false
     type: bool
+  liveserver:
+    description:
+      - This parameter was implemented a long time ago in a galaxy far way. It probably relates to the
+        django-liveserver package, which is no longer updated.
+      - Hence, it will be considered DEPRECATED and should be removed in a future release.
+    type: str
+    required: false
+    aliases: [live_server]
+  testrunner:
+    description:
+      - "From the Django docs: Controls the test runner class that is used to execute tests"
```

Comment:
```suggestion
      - "From the Django docs: Controls the test runner class that is used to execute tests".
```

Reply:
The full-stop should be inside the quotes. I'm fixing that on my side.